tests/tests/constants/testContracts.ts

```diff
@@ -190,3 +190,97 @@ export const FINITE_LOOP_CONTRACT_ABI = [
     type: "function",
   },
 ] as AbiItem[];
+
+export const ERC20_BYTECODE =
+  "0x608060405234801561001057600080fd5b50610024336202616061002960201b60201c565b610274565b600073ff" +
+  "ffffffffffffffffffffffffffffffffffffff168273ffffffffffffffffffffffffffffffffffffffff1614156100" +
```

Comment:
We should have the corresponding solidity (or however you wrote this contract) nearby. Some of the tests put it in a comment, or if this is an Open Zeppelin thing a link would be fine.